anitya/tests/request-data/anitya.tests.lib.backends.test_init.BaseBackendTests.test_expand_subdirs

```diff
@@ -0,0 +1,74 @@
+        Pkxhc3QgbW9kaWZpZWQ8L2E+PC90aD48dGg+PGEgaHJlZj0iP0M9UztPPUEiPlNpemU8L2E+PC90
+        aD48dGg+PGEgaHJlZj0iP0M9RDtPPUEiPkRlc2NyaXB0aW9uPC9hPjwvdGg+PC90cj48dHI+PHRk
+        IHZhbGlnbj0idG9wIj48aW1nIHNyYz0iL1RhbmdvLzI0eDI0L2FjdGlvbnMvZ28tdXAucG5nIiBh
+        bHQ9IltESVJdIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIC8+PC90ZD48dGQ+PGEgaHJlZj0iL3B1
+        Yi9saW51eC8iPlBhcmVudCBEaXJlY3Rvcnk8L2E+PC90ZD48dGQ+Jm5ic3A7PC90ZD48dGQgYWxp
+        Z249InJpZ2h0Ij4gIC0gPC90ZD48dGQ+IDwvdGQ+PC90cj4KPHRyPjx0ZCB2YWxpZ249InRvcCI+
+        PGltZyBzcmM9Ii9UYW5nby8yNHgyNC9wbGFjZXMvZm9sZGVyLnBuZyIgYWx0PSJbRElSXSIgd2lk
+        dGg9IjI0IiBoZWlnaHQ9IjI0IiAvPjwvdGQ+PHRkPjxhIGhyZWY9ImVuY2hpbGFkYS8iPmVuY2hp
+        bGFkYS88L2E+PC90ZD48dGQgYWxpZ249InJpZ2h0Ij4yMi1BdWctMjAxOCAxOTozMiAgPC90ZD48
+        dGQgYWxpZ249InJpZ2h0Ij4gIC0gPC90ZD48dGQ+IDwvdGQ+PC90cj4KPHRyPjx0ZCB2YWxpZ249
+        InRvcCI+PGltZyBzcmM9Ii9UYW5nby8yNHgyNC9wbGFjZXMvZm9sZGVyLnBuZyIgYWx0PSJbRElS
+        XSIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI0IiAvPjwvdGQ+PHRkPjxhIGhyZWY9ImVwZWwvIj5lcGVs
+        LzwvYT48L3RkPjx0ZCBhbGlnbj0icmlnaHQiPjIyLUF1Zy0yMDE4IDEzOjUxICA8L3RkPjx0ZCBh
+        bGlnbj0icmlnaHQiPiAgLSA8L3RkPjx0ZD4gPC90ZD48L3RyPgo8dHI+PHRkIHZhbGlnbj0idG9w
+        Ij48aW1nIHNyYz0iL1RhbmdvLzI0eDI0L3BsYWNlcy9mb2xkZXIucG5nIiBhbHQ9IltESVJdIiB3
+        aWR0aD0iMjQiIGhlaWdodD0iMjQiIC8+PC90ZD48dGQ+PGEgaHJlZj0ibGludXgvIj5saW51eC88
+        L2E+PC90ZD48dGQgYWxpZ249InJpZ2h0Ij4yMS1GZWItMjAxOCAyMToyMCAgPC90ZD48dGQgYWxp
+        Z249InJpZ2h0Ij4gIC0gPC90ZD48dGQ+IDwvdGQ+PC90cj4KPC90YWJsZT4KCQk8L2Rpdj4KCTxk
+        aXYgY2xhc3M9ImNsZWFyIj48L2Rpdj4KICAgICA8IS0tIGhvcml6b250YWxuaSBjYXJhIC0tPgog
+        ICAgICAgIDxwPgogICAgICAgIDxjb2RlPgogICAgICAgIGZ0cC1hZG1pbihhdClmaS5tdW5pLmN6
+        CiAgICAgICAgPCEtLSBuYXNsZWR1amljaSBhZHJlc2EgamUgcGFzdCBuYSBzcGFtLCBuZXBvc2ls
+        ZWp0ZSBuYSBuaSB6YWRuZSBlbWFpbHksIG5hIGt0ZXJlIGNoY2V0ZSBkb3N0YXQgb2Rwb3ZlZCAt
+        LT4KICAgICAgICA8YSBocmVmPSJtYWlsdG86aG51c0BmaS5tdW5pLmN6IiBjbGFzcz0iaGlkZGVu
+        Ij5obnVzQGZpLm11bmkuY3o8L2E+CiAgICAgICAgPC9jb2RlPgoJPGEgaHJlZj0iaHR0cDovL3d3
+        dy5saW51eC5jei9zdGF0cy8iPk8gc2VydmVydTwvYT4KICAgIDwvcD4KICAgIDwvZGl2PgogICAg
+        PC9kaXY+CiAgICA8L2Rpdj4KPCEtLQogICAgICAgIDxmb3JtIGFjdGlvbj0iaHR0cDovL2ZhZG1p
+        bi5maS5tdW5pLmN6L25vYXV0aC9jZ2ktYmluL3NlYXJjaC9zZWFyY2guY2dpIiBtZXRob2Q9Imdl
+        dCIgaWQ9InNlYXJjaCI+CiAgICAgICAgICA8aDI+PGEgaWQ9InNlYXJjaC1mb3JtIj48L2E+SGxl
+        ZOFu7TwvaDI+CiAgICAgICAgICA8cD4KICAgICAgICAgICAgPGlucHV0IHR5cGU9InRleHQiIG5h
+        bWU9InEiIGlkPSJ0ZXh0LWYiIG1heGxlbmd0aD0iMjU1IiAvPgogICAgICAgICAgICA8aW5wdXQg
+        dHlwZT0ic3VibWl0IiB2YWx1ZT0iaGxlZGF0IiBpZD0ic3VibWl0LWYiIC8+CiAgICAgICAgICA8
+        L3A+CiAgICAgICAgPC9mb3JtPgotLT4KPC9ib2R5PjwvaHRtbD4K
+    headers:
+      connection: [Keep-Alive]
+      content-type: [text/html;charset=UTF-8]
+      date: ['Thu, 23 Aug 2018 09:39:54 GMT']
+      keep-alive: ['timeout=5, max=100']
+      server: [Apache/2.2.22 (Fedora)]
+      x-clacks-overhead: [GNU Terry Pratchett]
```

Comment:
Nice

Reply:
I was surprised by this value when I fetched the URL.

Reply:
In case you're not familiar with Terry Pratchett's Discworld books, one of his books includes a lot of amusing references to the Free Software movement and this HTTP header is in [reference to one of his books](https://xclacksoverhead.org/home/about).

Reply:
I know his work. I read almost all his books from Discworld, but I was surprised to see the header field, I never saw it before.